simcon_project/users/forms.py

```diff
@@ -1,15 +1,46 @@
-from django.contrib.auth.forms import UserCreationForm, UserChangeForm
 from django import forms
+from django.contrib.auth.models import User
+from django.contrib.auth.forms import UserCreationForm, UserChangeForm
 from .models import CustomUser
+from users.models import Student, SubjectLabel
+import django_tables2 as tables
+
 
 
 class CustomUserCreationForm(UserCreationForm):
 
-    class Meta(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
         model = CustomUser
         fields = ('email',)
 
 
+class NewStudentCreationForm(forms.Form):
+    email = forms.EmailField(max_length=254, required=True)
+    first_name = forms.CharField(max_length=100, required=True)
+    last_name = forms.CharField(max_length=100, required=True)
+    password1 = forms.CharField(max_length=100, required=True)
```

Comment:
If you put in `password1 = forms.CharField(max_length=100, required=True, widget=forms.PasswordInput)` here then the field should just show the normal dots for each character.